scripts/delete_lambda_versions.py

```diff
@@ -0,0 +1,41 @@
+"""
+Delete all published AWS Lambda versions except for "$LATEST" in the current
+deployment.
+"""
+
+import logging
+
+from azul.deployment import (
+    aws,
+)
+from azul.lambdas import (
+    Lambdas,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    lambdas = Lambdas().list_lambdas()
+    for lambda_ in lambdas:
+        response = aws.lambda_.list_versions_by_function(
+            FunctionName=lambda_.name
+        )
+        for version in response['Versions']:
+            if version['Version'] == '$LATEST':
+                pass
+            else:
+                version_number = version['Version']
+                log.info('Deleting published version %s of %s', version_number, lambda_.name)
+                aws.lambda_.delete_function(
+                    FunctionName=lambda_.name,
+                    Qualifier=version_number
+                )
```

Comment:
```suggestion
            version = version['Version']
            if version == '$LATEST':
                pass
            else:
                log.info('Deleting published version %s of %s', version, lambda_.name)
                aws.lambda_.delete_function(
                    FunctionName=lambda_.name,
                    Qualifier=version
                )
```

Reply:
I think it's better to reuse the `version` variable instead of introducing a new one (especially since I'm confused about the difference in their names). Also, we can assign to it earlier to avoid repeating the dictionary lookup.